src/Components/AdvancedSearch/Internal/AdvancedSearchResultDetailsList/AdvancedSearchResultDetailsList.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { ComponentStory } from '@storybook/react';
+import { getDefaultStoryDecorator } from '../../../../Models/Services/StoryUtilities';
+import AdvancedSearchResultDetailsList from './AdvancedSearchResultDetailsList';
+import { IAdvancedSearchResultDetailsListProps } from './AdvancedSearchResultDetailsList.types';
+import { IADTTwin } from '../../../../Models/Constants';
+import twinData from '../../../../Adapters/__mockData__/MockAdapterData/DemoEnvsTwinData.json';
+
+const wrapperStyle = { width: '100%', height: '600px', padding: 8 };
+const filteredTwins: IADTTwin[] = twinData;
+
+const cols = [
+    {
+        key: `FailedPickupsLastHr + ${Math.random()}`,
+        name: 'Failed Pickups Last Hr',
+        fieldName: 'FailedPickupsLastHr',
+        minWidth: 100,
+        maxWidth: 200,
+        isResizable: true
+    },
+    {
+        key: `HydraulicPressure + ${Math.random()}`,
+        name: 'Hydraulic Pressure',
+        fieldName: 'HydraulicPressure',
+        minWidth: 100,
+        maxWidth: 200,
+        isResizable: true
+    }
+];
+export default {
+    title: 'Components/AdvancedSearchResultDetailsList',
```

Comment:
let's put these in a folder called "AdvancedSearch" and then you can group them under "Search Results"
`Components/Advanced Search/Search Results`